utility/src/main/java/org/oppia/android/util/profile/BUILD.bazel

```diff
@@ -20,5 +20,5 @@ kt_android_library(
     srcs = [
         "NameValidator.kt",
```

Comment:
Looks like this also needs to be renamed.

Reply:
done